.github/workflows/update-nix-inputs.yml

```diff
@@ -0,0 +1,18 @@
+name: Nix
+
+jobs:
+  update:
+    name: inputs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+      	uses: actions/checkout@v3
+
+      - uses: DeterminateSystems/nix-installer-action@main
+      - name: Update inputs
+	      run: nix/update-inputs.sh
```

Comment:
Seems like we need that script in the repo, heres Hyprland's script: https://github.com/hyprwm/Hyprland/blob/main/nix/update-inputs.sh

i know nothing about nix so I can't help on how we would adapt it though

Reply:
As it is just needed to update the inputs without checking, I've used the justfile.